with-router-typescript/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "scripts": {
+    "start": "expo start"
+  },
+  "dependencies": {
+    "@types/react": "~18.0.27",
+    "expo": "^48.0.7",
+    "expo-constants": "~14.2.1",
+    "expo-linking": "~4.0.1",
+    "expo-router": "^1.2.0",
+    "expo-splash-screen": "~0.18.1",
+    "expo-status-bar": "~1.4.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.71.3",
+    "react-native-gesture-handler": "~2.9.0",
+    "react-native-reanimated": "~2.14.4",
+    "react-native-safe-area-context": "4.5.0",
+    "react-native-screens": "~3.20.0",
+    "react-native-web": "~0.18.7",
+    "typescript": "^4.9.4"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.19.3",
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9"
+  },
+  "resolutions": {
+    "metro": "^0.73.7",
+    "metro-resolver": "^0.73.7"
+  },
+  "overrides": {
```

Comment:
why do we need it?

Reply:
`"resolutions"` and `"overrides"` are in the original `package.json` from `with-router` as well:

https://github.com/expo/examples/blob/bd68019bc808479a3ff99d0a6f69374871713b21/with-router/package.json#L25-L32

probably some compatibility thing